doc/ansible/dev/developer_guide.md

```diff
@@ -470,6 +470,9 @@ The structure passed to Ansible as extra vars is:
   "_app_example_com_database": {
      <Full CRD>
    },
+  "_app_example_com_database_spec": {
```

Comment:
perhaps we should make this a more easily usable variable? The `_<group>_<kind>` pattern isn't especially friendly and is something that we've discussed changing in the past. @chancez any opinion on that? I don't want to hold up your PR with random discussion but thought it was worth mentioning before doubling down on the pattern.

Reply:
@shawn-hurley ^

Reply:
I can see it both ways. I think namespacing variables is important, and in practice, all my default vars are already prefixed with "meteringconfig_" already, so having something like `_<kind>_spec` would probably be acceptable, but I think the `_<group>_<kind>` isn't terrible. It's easy enough to alias to a different variable in `defaults/`, which is what I do.

Reply:
@fabianvf I say we create an issue and continue the discussion over there if you feel strongly. 

I personally don't have strong feelings but think it makes sense to leave as is, making it non-obvious and namespaced/hidden make it so that it is not overriding variables the user would have made a separate variable name.

Reply:
nah just thought I'd bring it up since we have had the discussion, happy to close discussion now